engine/src/main/java/org/terasology/config/AudioConfig.java

```diff
@@ -31,39 +31,63 @@
 
     private transient PropertyChangeSupport propertyChangeSupport = new PropertyChangeSupport(this);
 
+    /**
+     * @return Returns a float of the sound volume
```

Comment:
I'd actually phrase it as:
@return the sound volume as a float, with 0 meaning no sound and 100 (or 1?) being the maximum volume.